yamcs-core/src/main/java/org/yamcs/tctm/ccsds/UdpTmFrameLink.java

```diff
@@ -111,16 +109,10 @@ public String getDetailedStatus() {
     public Map<String, Object> getExtraInfo() {
         var extra = new LinkedHashMap<String, Object>();
         extra.put("Valid datagrams", frameCount.get());
-        extra.put("Invalid datagrams", invalidDatagramCount);
```

Comment:
Thanks for your pull request.

Shouldn't we call this "Valid frames" and "Invalid frames"?

Reply:
Good point, I added a commit for this.